onelogin_rules/onelogin_password_changed.yml

```diff
@@ -0,0 +1,50 @@
+AnalysisType: rule
+Filename: onelogin_password_changed.py 
+RuleID: OneLogin.PasswordChanged
+DisplayName: OneLogin User Password Changed
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+Tags:
+  - OneLogin
+  - Identity & Access Management
+Severity: Medium
```

Comment:
I might put this at a `Low`, what're your thoughts @lindsey-w ?

Reply:
I guess I was separating this from "Admin changed a user's password" (event 211). Don't know why they explicitly separated this into two different event types...so in this rule - its the case that `actor_user is not an admin` and `actor_user != user` for a password change event. In my mind, this might be a more severe than a user changing their own password or an admin changing a user's password.  Perhaps this would never happen? I could change it to not care if `actor_user == user` and make it a lower severity?

Reply:
Ah I see, I didn't realize that this was specifically a non-admin user changing another user's password. I also don't know if this is possible, I would suggest removing the `actor_user != user` condition and just making this an `Info`.